src/transactions/database.js

```diff
@@ -29,6 +30,16 @@ export const createDatabase = async (connection, dbName, done) => {
   return false
 }
 
+/**
+ * connection database
+ */
+export const getConnection = async (database = '') =>
+  createLink({
+    host: process.env.ENV === 'mock' ? '172.18.0.2' : 'localhost',
+    port: process.env.ENV === 'mock' ? '28015' : '55001',
```

Comment:
I really think that this port should be fixed as 28015 and the fix should be done in the docker container mapping correctly the port exposure